client/src/components/Review Widget/ProductBreakDownBar.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+const ProductBreakDownBar = (props) => {
+  //infile styling required to place thing over bars.
+
+  const charBar = {
+    height: 10 + 'px',
+    background: 'lightgrey',
+    width: 32.5  + '%',
+  }
+  const charRating = {
+    height: 10 + 'px',
+    width: 3 + 'px',
+    left: ((props.rating/5) * 257) + 'px',
+    position: 'absolute',
+    background: 'green',
+  }
+
+  return (
+    <>
```

Comment:
Just nit picking. You already have a wrapping div here so you can get rid of this fragment